src/config/rcxml.c

```diff
@@ -317,6 +317,26 @@ get_accel_profile(const char *s)
 	return LIBINPUT_CONFIG_ACCEL_PROFILE_FLAT;
 }
 
+static enum libinput_config_send_events_mode
+get_send_events_mode(const char *s)
+{
+	if (!s) {
+		wlr_log(WLR_INFO, "Not a recognised node.");
```

Comment:
s/node/mode/

Also, I suggest using ERROR instead of INFO for this to alert the user without them having to run in info/debug mode

Reply:
and we might want to change the message to something like "Not a recognized mode for libinput sendEventsMode".
Otherwise a user seeing that error message in the logs won't necessarily know what that error is about.